fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1322,6 +1515,29 @@ public Builder setRecordTypes(@Nullable Collection<RecordType> recordTypes) {
             return this;
         }
 
+        /**
+         * Get the function used by the online indexer to load the config parameters on fly.
```

Comment:
Probably should add that the config loader will be given the current configuration as input at the beginning of each transaction and should produce the configuration to use in the next transaction.

Or this information should go in the `setConfigLoader` function.

Reply:
sure, put in `set`